webcomponents/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="./dist/index.mjs" type="module"></script>
+    <link rel="stylesheet" href="./dist/style.css" />
+    <title>JoyRTC Test</title>
+    <style>
+      #root {
+        max-width: 1280px;
+        margin: 0 auto;
+        padding: 2rem;
+        text-align: center;
+      }
+    </style>
+  </head>
+  <body>
+    <div id="root">
+      <joy-rtc style="width: 1024"></joy-rtc>
```

Comment:
This `<joy-rtc>` element does not have a `address=` attribute, so it does not know the WebSocket URL to connect to. Since the WebSocket URL depends on URL of the page, maybe it's better to set the address in `index.ts` (see below) rather than hardcode it there.